traefik/templates/_podtemplate.tpl

```diff
@@ -640,32 +643,32 @@
              {{- end }}
              {{- range $field, $value := .metrics.opentelemetry.tls }}
               {{- if has $field (list "ca" "cert" "insecureSkipVerify" "key") -}}
-               {{ with $value }}
+               {{- with $value }}
           - "--hub.metrics.opentelemetry.tls.{{ $field }}={{ $value }}"
                {{- end }}
               {{- end }}
              {{- end }}
             {{- end }}
-            {{ with .platformurl }}
-          - "--hub.platformurl={{ . }}"
+            {{- with .platformUrl }}
```

Comment:
This was correct. This should be aligned with command line arguments.

Reply:
All parameters of Traefik are camel cased in the documentation